### PR TITLE
Entity and relationship improvements

### PR DESCRIPTION
**Pull request by fomentia**
@erkangz I think this gets everything that you wrote in the Jira ticket.

### CODE REVIEW

**Comment by fomentia** [comment]
Oops. It gets everything in the Jira ticket EXCEPT for what's in the title of the Jira ticket. 😆 Lemme get those rq...

**Comment by fomentia** [comment]
Jira ticket link for @aiwilliams: https://lifeomic.atlassian.net/browse/JPT-1901

**Comment by fomentia** [comment]
It doesn't look like Veracode provides a link back to the web for findings. ☹️ 

This PR is ready for review now.

**Comment by fomentia** [comment]
Actually, I think I am wrong about the webLink thing. BRB

File: src/converters.test.ts [hunk 1]
```diff
@@ -69,5 +91,27 @@ test("convert findings -> finding entities and date type properties -> number ty
     sourceFilePath: "file_path",
     sourceModule: "module",
     targets: "name",
+    webLink: '{"Link 1":"asdf","Link 2":"hjkl"}',
```
Comment (fomentia) [inline src/converters.test.ts]: This obviously isn't what we want it to be forever. However, there's no way to tell from the [Veracode schema docs](https://help.veracode.com/viewer/book-attachment/LMv_dtSHyb7iIxAQznC~9w/6UifnFyJMsCcnh9GhHayWw) what exactly will be included in the links property, as it's an array of objects. 😕 I figure I need to see what's there using ClearCare, and then update this integration.

Comment (erkangz) [reply src/converters.test.ts]: how about `webLink: links[0].href`, `webLink1: links[1].href`, etc....

File: src/converters.ts [hunk 2]
```diff
@@ -124,6 +132,31 @@ export function toCWEEntity(finding: FindingData): CWEEntity {
...
+ * working.
+ */
+
+const severityMap: { [numericSeverity: number]: string } = {
+  0: "Informational",
+  1: "Very Low",
+  2: "Low",
+  3: "Medium",
+  4: "High",
+  5: "Very High",
```
Comment (erkangz) [inline src/converters.ts]: let's make this "Critical" to match the rest in J1

File: src/converters.test.ts (hunk 1 above)
Comment (fomentia) [reply src/converters.test.ts]: @erkangz Excellent idea. Will do rnrq

File: src/converters.ts [hunk 3]
```diff
@@ -124,6 +132,31 @@ export function toCWEEntity(finding: FindingData): CWEEntity {
...
+ * For more information on these mappings, check out the Veracode documentation:
+ * https://help.veracode.com/reader/DGHxSJy3Gn3gtuSIN2jkRQ/y6AoBBzDtboSZ~nOUsQUDg
+ *
+ * Please note that we cannot guarantee that the above docs link will continue
+ * working.
+ */
+
+const severityMap: { [numericSeverity: number]: string } = {
+  0: "Informational",
+  1: "Very Low",
```
Comment (erkangz) [inline src/converters.ts]: I think it's fine to have 1 and 2 both be "Low" - this will match the rest of J1

**Pull request merged**